tests/FSharp.Compiler.Service.Tests/ConsoleOnlyOptionsTests.fs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information.
+
+module FSharp.Compiler.Service.Tests.ConsoleOnlyOptionsTests
+
+open System
+open System.IO
+open FSharp.Compiler.Text
+open FSharp.Compiler.AbstractIL.ILBinaryReader
+open FSharp.Compiler.CompilerConfig
+open FSharp.Compiler.CompilerOptions
+open Internal.Utilities
+open NUnit.Framework
+
+// just a random thing to make things work
+let private getBuilder() =
+    TcConfigBuilder.CreateNew(
```

Comment:
Put this in the shared unit test utilities, and call it something like GetTcConfigBuilderForUnitTesting or GetArbitraryTcConfigBuilder.  We don't want copied of this scattered around :)

Reply:
Yep, done